### PR TITLE
[POT] Remove tpe algo

### PR DESCRIPTION
**Pull request by MilanaShhanukova**
### Tickets:
 - CVS-68236

### CODE REVIEW

**Comment by AlexKoff88** [comment]
@ematroso, @evgenytalanin-intel, we are going to remove TPE method. So, no validation is expected for that feature afterwards.

**Review by AlexKoff88 — APPROVED** [review]

File: tools/pot/docs/BestPractices.md [hunk 1]
```diff
@@ -124,151 +124,5 @@ accuracy-aware scenario.
 
 If you do not achieve the desired accuracy and performance after applying the 
 `AccuracyAwareQuantization` algorithm or you need an accurate fully-quantized model,
-we recommend either using layer-wise hyperparameters tuning with TPE or using 
+we recommend either using layer-wise hyperparameters tuning using
```
Comment (MaximProshin) [inline tools/pot/docs/BestPractices.md:127]: 'either' is unnecessary here

**Review by MaximProshin — APPROVED** [review]

**Review by KodiaqQ — APPROVED** [review]

**Comment by ematroso** [comment]
> @ematroso, @evgenytalanin-intel, we are going to remove TPE method. So, no validation is expected for that feature afterwards.

TPE validation has been already disabled on our side.

File: tools/pot/docs/BestPractices.md (hunk 1 above)
Comment (MilanaShhanukova) [reply tools/pot/docs/BestPractices.md]: Thanks. Didn't notice.

**Review by avladimi — COMMENTED** [review]
Reviewed, approved

**Pull request merged**